sql/database.sql

```diff
@@ -3537,7 +3537,8 @@ CREATE TABLE `lang_definitions` (
   `lang_id` int(11) NOT NULL default '0',
   `definition` mediumtext,
   UNIQUE KEY `def_id` (`def_id`),
-  KEY `cons_id` (`cons_id`)
+  KEY `cons_id` (`cons_id`),
+  KEY `lang_cons` (`lang_id`, `cons_id`)
```

Comment:
hi @kojiromike ,
It just hit me that this will not work for the translation table since it's overwritten by the language import (note the upgrade stuff in the upgrade script will work).
Definitely need to keep this change for clarity sake, but will need changes in 3 other places to ensure persistence:
1. lang_definitions table in https://raw.githubusercontent.com/openemr/openemr/refs/heads/master/contrib/util/language_translations/currentLanguage_utf8.sql
2. https://github.com/openemr/openemr/blob/master/contrib/util/language_translations/buildLanguageDatabase.pl#L554-L555
3. lang_definitions table in rel-704 (same file as step 1 above, but in rel-704 branch; this is important since we will likely be copying this over for 8.0.0 release)